src/ConnectedMode/Migration/Wizard/MigrationWizardWindow.xaml.cs

```diff
@@ -46,9 +57,19 @@ private void NavigateToMigrationProgressPage(object sender, RoutedEventArgs e)
 
         private void OnStartMigration(object sender, RoutedEventArgs e)
         {
+            if (migrationInProgress) { return; }
```

Comment:
I added this simply to ensure that OnStartMigration can never be executed if it is already running